phpBB/includes/ucp/ucp_pm_compose.php

```diff
@@ -978,7 +978,26 @@ function compose_pm($id, $mode, $action, $user_folders = array())
 			censor_text($message_parser->message),
 			$quote_attributes
 		);
-		$message_parser->message = $message_link . $quote_text . "\n\n";
+		if ($bbcode_status)
+		{
+			$message_parser->message = $message_link . $quote_text . "\n\n";
+		}
+		else
+		{
+			$offset = 0;
```

Comment:
IMHO this code should be extracted to somewhere else to avoid duplication with the one taken from `posting.php`.
It can possibly be useful for other types of messages (e.g. introduced by extension developers).